MAUI/DataForm/Data-annotations.md

```diff
@@ -0,0 +1,494 @@
+
+## ReadOnly attribute
+
+It specifies whether the data field is read only or not.
+
+{% tabs %}
+{% highlight ReadOnly attribute}
+
+    [ReadOnly(false)]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+
+
+## DataType attribute
+
+It specifies data type for the field.
+
+Supported data types are Text, MultilineText, Date, DateTime and Time.
+
+{% tabs %}
+{% highlight DataType attribute}
+
+    [DataType(DataType.MultilineText)]
+    public string Address { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Custom attribute
+The data form supports the following custom attribute, and these attributes can be accessible using `Syncfusion.Maui.DataForm`.
+
+### DataFormDisplayOptions attribute
+
+<table>
+<tr>
+<th>
+Property
+</th>
+<th>
+Details
+</th>
+</tr>
+<tr>
+<td>
+[RowSpan]
+</td>
+<td>
+Specifies the row span for the data form item.
+</td>
+</tr>
+<tr>
+<td>
+[ColumnSpan]
+</td>
+<td>
+Specifies the column span for the data form item.
+</td>
+</tr>
+<tr>
+<td>
+[ValidMessage]
+</td>
+<td>
+Specifies positive message to be shown when validation is passed. 
+</td>
+</tr>
+<tr>
+<td>
+[ShowLabel]
+</td>
+<td>
+Specifies whether the label should be visible or not.
+</td>
+</tr>
+<tr>
+<td>
+[ItemsOrderInRow]
+</td>
+<td>
+Specifies the ItemsOrderInRow property to display multiple editors in a single row. Two items with the same RowOrder property value are placed in this row according to their ItemOrderInRow property values.
+</td>
+</tr>
+</table>
+
+{% tabs %}
+{% highlight RowSpan attribute}
+
+    [DataFormDisplayOptions(RowSpan =2)]
+    public string Address { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight ColumnSpan attribute}
+
+    [DataFormDisplayOptions(ColumnSpan =2)]
+    public string Name { get; set; }    
+
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight ValidMessage attribute}
+
+    [StringLength(20, ErrorMessage = "Value should not empty")]
+    [DataFormDisplayOptions(ValidMessage = "Text length is enough")]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight ShowLabel attribute}
+
+    [DataFormDisplayOptions(ShowLabel = false)]
+    public string City { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight ItemsOrderInRow attribute}
+
+    dataForm.columnCount = 3;
+    
+    [DataFormDisplayOptions(ItemsOrderInRow =2)]
+    public string Address { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+### DataFormValueConverter attribute
+
+<table>
+<tr>
+<th>
+Property
+</th>
+<th>
+Details
+</th>
+</tr>
+<tr>
+<td>
+[DataFormValueConverterType]
+</td>
+<td>
+Specifies the Converter type which converts the original value in different format or as different value.
+</td>
+</tr>
+</table>
+
+{% tabs %}
+{% highlight  attribute}
+
+    [DataFormValueConverter(typeof(StringToStringConverter))]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
```

Comment:
Refer to this [link] to know more about `DataFormValueConverterAttribute`

Reply:
above mentioned line added